app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.author = current_user
+    @comment.post = Post.find(params[:post_id])
+
+    respond_to do |_format|
+      if @comment.save
+
+        redirect_to user_post_path(user_id: params[:user_id], id: params[:post_id])
```

Comment:
- Good job done so far, but this does not redirect to the user_post_path` when a new comment is created, please fix this 🙏